pkg/advisory/create.go

```diff
@@ -55,7 +58,7 @@ func Create(ctx context.Context, req Request, opts CreateOptions) error {
 		})
 		err := documents.Update(ctx, u)
 		if err != nil {
-			return fmt.Errorf("unable to create advisory %q for %q: %w", newAdvisoryID, req.Package, err)
+			return fmt.Errorf("unable to create advisory %q for %q: %w", req.Aliases[0], req.Package, err)
```

Comment:
Ditto comment above re: `req.Aliases[0]`, we shouldn't give preference to one alias over others. Here, we could simply show the entire slice.

Reply:
Sorry, just to clarify — `req.Aliases[0]` isn't the entire slice. We can leave this for a later PR if you want just wanted to close the loop!

Reply:
done to show the entire slice